deeds-dapp-webapp/src/main/webapp/WEB-INF/jsp/placeholder.jsp

```diff
@@ -1,50 +1,51 @@
 <%@page import="io.meeds.deeds.web.utils.Utils"%>
 <%
   String buildnumber = Utils.getApplicationBuildNumber();
+  String basePath = request.getServletPath().startsWith("/fr/") ? "../" : "./";
```

Comment:
No need to make this change on placeholder

```suggestion
```

Reply:
done!